src/cool_seq_tool/sources/uta_database.py

```diff
@@ -10,16 +10,16 @@
 import asyncpg
 import boto3
 import polars as pl
+from agct import Converter
 from asyncpg.exceptions import InterfaceError, InvalidAuthorizationSpecificationError
 from botocore.exceptions import ClientError
-from pyliftover import LiftOver
 
 from cool_seq_tool.schemas import AnnotationLayer, Assembly, Strand
 
 # use `bound` to upper-bound UtaDatabase or child classes
 UTADatabaseType = TypeVar("UTADatabaseType", bound="UtaDatabase")
 
-# Environment variables for paths to chain files for pyliftover
+# Environment variables for paths to chain files for agct
 LIFTOVER_CHAIN_37_TO_38 = environ.get("LIFTOVER_CHAIN_37_TO_38")
 LIFTOVER_CHAIN_38_TO_37 = environ.get("LIFTOVER_CHAIN_38_TO_37")
```

Comment:
@korikuzma are these still used? `agct` should just be pulling them via wags-tails

Reply:
Ooooh, I think we need to provide support to agct to allow env vars to be passed for the chain files. I plan on storing these files in EFS and pointing to that mounted location

Reply:
You should be able to set `WAGS_TAILS_DIR` to a parent directory, if that helps.

Reply:
@jsstevenson What if we want to bypass wags-tails all together?

Reply:
@jsstevenson This is what I would propose:

Current:

```python
    def __init__(self, from_db: Genome, to_db: Genome) -> None:
        ....
        data_handler = CustomData(
            f"chainfile_{from_db.value}_to_{to_db.value}",
            "chain",
            lambda: "",
            self._download_function_builder(from_db, to_db),
            data_dir=get_data_dir() / "ucsc-chainfile",
        )
        file, _ = data_handler.get_latest()
```

Suggested change:

```python
    def __init__(self, from_db: Genome, to_db: Genome, file: Optional[Path] = None) -> None:
        ...
        if not file:
            data_handler = CustomData(
                f"chainfile_{from_db.value}_to_{to_db.value}",
                "chain",
                lambda: "",
                self._download_function_builder(from_db, to_db),
                data_dir=get_data_dir() / "ucsc-chainfile",
            )
            file, _ = data_handler.get_latest()
```

Reply:
yeah, I think so -- make a PR?

Reply:
Cool, wanted to make sure I wasn't missing something before making changes.